### DISCUSSION TITLE
Optimizing Turbo Mode: Balancing Performance and Temperature in ASUS G14 Gaming Laptops (help pls...)

### DISCUSSION THREAD

**Discussion by MrGoodo** [seerge/g-helper#2284] [Q&A]
Hello, I'm asking for help from experts. I have an ASUS G14 laptop (2022) with a Radeon CPU 6900HS and RX 6800S. Please assist me with setting up the turbo mode for gaming. With the help of this program, I managed to lower the temperatures by configuring turbo mode settings as shown in the screenshot. However, I turned off turbo boost CPU, and now I'm concerned whether I've lost a significant amount of performance because of this. I've watched many videos where people disable this setting, but some videos with ASUS laptops still leave it enabled for turbo mode. However, if I leave undervolt as it is now and turn on CPU boost, the laptop immediately starts to heat up. Therefore, my question is, can someone help me configure the turbo mode of my laptop for better gaming performance without reaching temperatures above 100 degrees?
![image](https://github.com/seerge/g-helper/assets/110602771/93e30c2c-eb77-445f-93ea-b896c0126122)
![image](https://github.com/seerge/g-helper/assets/110602771/84be8775-52dd-41d5-9680-8efd90249f91)

**Comment by Akalikaton** [comment]
I think first of all you need calibrate your fans. So you can get more than 7000RPM with full Perfermance.
<img width="500" alt="image" src="https://github.com/seerge/g-helper/assets/127612519/5d67b379-ac79-4959-8eb5-6cfefaf6ee86">

**Reply by MrGoodo** [reply]
I don't know why, but the maximum speed that the fans reach after calibration is 5800.
![image](https://github.com/seerge/g-helper/assets/110602771/a73457a7-152c-4fcb-b334-fe2124a12bd2)
